lib/sscanf.c

```diff
@@ -1,21 +1,7 @@
+// SPDX-License-Identifier: BSD-3-Clause
 /*
- ****************************************************************************
- *
- *        File: printf.c
- *      Author: Juergen Gross <jgross@suse.com>
- *
- *        Date: Jun 2016
- *
- * Environment: Xen Minimal OS
- * Description: Library functions for printing
- *              (FreeBSD port)
- *
- ****************************************************************************
- */
-
-/*-
  * Copyright (c) 1990, 1993
- *	The Regents of the University of California.  All rights reserved.
+ * The Regents of the University of California. All rights reserved.
```

Comment:
You can still put Juergen's authorship here. The original comment was to remove stars and have the comments U-boot style. This doesn't mean you have to drop the contents of what was commented